go.sum

```diff
@@ -605,6 +605,8 @@ honnef.co/go/tools v0.0.0-20190106161140-3f1c8253044a/go.mod h1:rf3lG4BRIbNafJWh
 istio.io/api v0.0.0-20190515205759-982e5c3888c6/go.mod h1:hhLFQmpHia8zgaM37vb2ml9iS5NfNfqZGRt1pS9aVEo=
 istio.io/api v0.0.0-20190802144653-982935ba456e h1:h+7PT/63ylxoseX8SMw8XxvsyVexDBmySZvPRVMDppA=
 istio.io/api v0.0.0-20190802144653-982935ba456e/go.mod h1:42cBjnu/rTJcCaKi8nLdIvq0n71RcLrkgZ9IQSvDdSQ=
```

Comment:
Seems like `go mod tidy` wasn't run. We really need a linter for this though..

https://github.com/istio/istio/wiki/Vendor-FAQ

Reply:
Ack, thanks for the link! Done.